wee_slack.py

```diff
@@ -2605,6 +2605,23 @@ def process_message(message_json, eventrouter, store=True, **kwargs):
             channel.store_message(message, team)
         dbg("NORMAL REPLY {}".format(message_json))
 
+    download_location = config.get_string('files_download_location')
```

Comment:
Can you place this (everything you're adding in this function) in a separate function?